config/debian/dpkg_client/debian/control

```diff
@@ -8,6 +8,6 @@ Homepage: https://github.com/google/grr
 
 Package: %(ClientBuilder.package_name)
 Architecture: any
-Depends: lsb-base \(>= 3.2-14\), ${shlibs:Depends}, ${misc:Depends}
+Depends: lsb-base \(>= 3.2-14\), libc6 \(>= 2.11\)
```

Comment:
FTR I'm going to remove this libc hard-coded dependency since it will be confusing if people build on older systems.  debhelper adds lots of unnecessary dependencies, so unpicking all of them except the libc dependency seems hacky.  The rule is basically that you build on a system with the oldest libc you want to support.